extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -76,7 +76,8 @@ export class AmpAd extends AMP.BaseElement {
 
       // TODO(tdrl): Check amp-ad registry to see if they have this already.
       if (!a4aRegistry[type] ||
-          this.win.document.querySelector('meta[name=amp-3p-iframe-src]') ||
+          (this.win.document.querySelector('meta[name=amp-3p-iframe-src]') &&
```

Comment:
Not a fan of the fact that RTC (which is doubleclick specific) is cluttering up amp-ad.  We should rethink how to do this later...